parcels/kernel.py

```diff
@@ -178,15 +170,23 @@ def __init__(self, fieldset, ptype, pyfunc=None, funcname=None, funccode=None, p
         else:
             self.funcvars = None
         self.funccode = funccode or inspect.getsource(pyfunc.__code__)
+        self.funccode = (  # Remove parcels. prefix (see #1608)
+            self.funccode
+            .replace('parcels.rng', 'rng')
+            .replace('parcels.ParcelsRandom', 'ParcelsRandom')
+            .replace('parcels.StatusCode', 'StatusCode')
+        )
+
         # Parse AST if it is not provided explicitly
-        self.py_ast = py_ast or parse(BaseKernel.fix_indentation(self.funccode)).body[0]
+        self.py_ast = py_ast or parse(textwrap.dedent(self.funccode)).body[0]  # Dedent allows for in-lined kernel definitions
```

Comment:
Why this new method? What was wrong with the old one?

Reply:
`BaseKernel.fix_indentation()` does the exact same as `textwrap.dedent()`. For maintainability and documentation, I would prefer to rely on standard library code rather than writing custom code.

As an aside I think simple string utilities like this shouldn't be included as public static methods, but should instead be private utility functions. `BaseKernel.fix_indentation()` has no purpose in the public API